src/main/java/org/jabref/logic/importer/fileformat/PdfContentImporter.java

```diff
@@ -364,12 +367,14 @@ Optional<BibEntry> getEntryFromPDFContent(String firstpageContents, String lineS
         String volume = null;
         String number = null;
         String pages = null;
+        String arXivId = null;
         // year is a class variable as the method extractYear() uses it;
         String publisher = null;
 
         EntryType type = StandardEntryType.InProceedings;
         if (curString.length() > 4) {
             // special case: possibly conference as first line on the page
+            arXivId = getArXivId(null);
```

Comment:
This is an example why comments are to be avoided as much as possible, this line compiles fine, runs fine but the comment above it is misguiding. Please move comment to its correct location.